relay/pkg/node/node.go

```diff
@@ -1,20 +1,56 @@
 package node
 
 import (
+	"context"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/tbtc/relay/pkg/block"
 	"github.com/keep-network/tbtc/relay/pkg/btc"
 	"github.com/keep-network/tbtc/relay/pkg/chain"
-	"github.com/keep-network/tbtc/relay/pkg/forwarder"
 )
 
+var logger = log.Logger("relay-node")
+
 // Initialize initializes the relay node.
+//
+// TODO: This function will be probably the right place to handle new
+//  requirements which will require starting and stopping the block forwarder.
```

Comment:
Agreed. 😆  Though this is already happening, it's not a TODO?

Reply:
It's not done as we don't support new features yet. What we have done is just forwarder which can be disabled using contexts and is restarted upon failures.

Reply:
Ah, I see - we are talking about relay auctions. Yes, it makes sense. Let's make it clear in the TODO what we are talking about. Proposition:
```
// TODO: This function will be probably the right place to handle relay auctions
// which will require starting and stopping the block forwarder.
```

Reply:
See https://github.com/keep-network/tbtc/commit/551cf7c8e780374c860a4edc9c286579705867f7